itl/command-plugins-manubulon.conf

```diff
@@ -215,6 +215,10 @@ object CheckCommand "snmp-storage" {
 			value = "$snmp_storage_olength$"
 			description = "Max-size of the SNMP message, usefull in case of Too Long responses."
 		}
+    "-q" = {
```

Comment:
Please indent following the rest of the file (tabs)

Reply:
Fixed indents and updated docs.